.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,61 @@
+name: Run tests in Centos container
+
+on: [pull_request, push]
+
+jobs:
+  tests-centos-8:
+    name: "Unittests on Centos 8"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Run in container"
+        uses: addnab/docker-run-action@v3
+        with:
+          image: quay.io/osbuild/osbuild-ci-c8s
```

Comment:
AFAICT the code for both distros is identical. There are two ways to avoid duplication.
1. Create a bash script and just call that from both functions.
2. Use the [matrix strategy](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix).

I think we use either of these approaches throughout our repositories, so either would work. I would probably lean towards 2 because it's more compact and the action remains more self-contained.

Reply:
Agree with option two as I saw it already in use somewhere:)

Reply:
Done